README.markdown

```diff
@@ -16,7 +16,7 @@ Visit the [project page](http://benalman.com/projects/jquery-hashchange-plugin/)
 These working examples, complete with fully commented code, illustrate a few
 ways in which this plugin can be used.
 
-[http://benalman.com/code/projects/jquery-hashchange/examples/hashchange/](http://benalman.com/code/projects/jquery-hashchange/examples/hashchange/)  
```

Comment:
@cowboy Actually, what semantic meaning were you after here?

I can't spot an issue with the rendering:
https://github.com/cowboy/jquery-hashchange/blob/master/README.markdown#examples
![screen shot 2015-03-26 at 07 52 07](https://cloud.githubusercontent.com/assets/156867/6842227/4ef44a3a-d38d-11e4-8038-95081104b09b.png)

https://github.com/Krinkle/jquery-hashchange/blob/cleanup/README.markdown
![screen shot 2015-03-26 at 07 51 44](https://cloud.githubusercontent.com/assets/156867/6842228/4ef635c0-d38d-11e4-990d-5c5f9993d9e2.png)

----

Googling shows it may relate inserting line breaks, but at least in the GitHub flavour of Markdown that has always worked naturally by using a regular single line break. It doesn't collapse linebreaks or require two line breaks for a paragraph.

```
Foo
Bar

Quux
``` 
> Foo
> Bar
> 
> Quux

```html
<p>Foo<br>
Bar</p>
<p>Quux</p>
```